nomad/structs/services_test.go

```diff
@@ -8,6 +8,113 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func TestService_Hash(t *testing.T) {
+	t.Parallel()
+
+	original := &Service{
+		Name:      "myService",
+		PortLabel: "portLabel",
+		// AddressMode: "bridge", // not hashed
```

Comment:
Maybe explain why since it's such a weird field.

```suggestion
		// AddressMode: "bridge", // not hashed (used internally by Nomad)
```